lib/cdk-infra-stack.ts

```diff
@@ -501,14 +514,37 @@ export default class FhirWorksStack extends Stack {
             description: 'FHIR API Server',
             entry: path.join(__dirname, '../src/index.ts'),
             handler: 'handler',
-            bundling: defaultLambdaBundlingOptions,
+            bundling: {
+                ...defaultLambdaBundlingOptions,
+                commandHooks: {
+                    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                    beforeBundling(inputDir, outputDir) {
+                        return [];
+                    },
+                    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                    beforeInstall(inputDir, outputDir) {
+                        return [];
+                    },
+                    afterBundling(inputDir, outputDir) {
+                        // copy all the necessary files for the lambda into the bundle
+                        // this allows the lambda functions for bulk export to have access to these files within the lambda instance
+                        return [
+                            `cp -r ${inputDir}\\compiledImplementationGuides ${outputDir}`,
+                        ];
+                    },
+                },
+            },
             runtime: Runtime.NODEJS_14_X,
-            reservedConcurrentExecutions: 5,
+            currentVersionOptions: {
+                provisionedConcurrentExecutions: 5,
+            },
             environment: {
                 ...lambdaDefaultEnvVars,
                 EXPORT_STATE_MACHINE_ARN: bulkExportStateMachine.bulkExportStateMachine.stateMachineArn,
                 PATIENT_COMPARTMENT_V3,
                 PATIENT_COMPARTMENT_V4,
+                // config.ts only checks if the arn is present or equal to [object Object]
+                VALIDATOR_LAMBDA_ALIAS: props!.useHapiValidator ? this.javaHapiValidator!.hapiValidatorLambda.functionArn : '[object Object]',
```

Comment:
Is there a way to update the config.ts check so there's no need to assign '[object Object]' to ALIAS value? It looks quite hacky.

Reply:
I changed it to also check for empty strings so we could use that instead. Unfortunately, i wasn't able to use `undefined`, so i had to use a string of some sort since that is the type it was expecting.

Reply:
Empty string works! Thanks for diving in this one.